From 4ba9d2f7613a39d3a8db4883e6e806f47896c140 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Thu, 10 Jul 2014 01:07:03 -0400
Subject: [acpi] memhotplug: Use defined marco METHOD_NAME__STA

Message-id: <20140710010703.22146.71748.stgit@gir.stowe>
Patchwork-id: 85624
O-Subject: [RHEL7.1 PATCH 085/105] ACPI / memhotplug: Use defined marco METHOD_NAME__STA
Bugzilla: 1114228
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1114228
Upstream Status: 16ff816d3b5d2b81fcff5ca44eb9a98ac3b604b4

commit 16ff816d3b5d2b81fcff5ca44eb9a98ac3b604b4
Author: Zhang Yanfei <zhangyanfei@cn.fujitsu.com>
Date:   Wed Oct 2 16:27:37 2013 +0800

    ACPI / memhotplug: Use defined marco METHOD_NAME__STA

    We already have predefined marco for method name "_STA', so
    using the marco instead of directly using the string.

    Signed-off-by: Zhang Yanfei <zhangyanfei@cn.fujitsu.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/acpi/acpi_memhotplug.c b/drivers/acpi/acpi_memhotplug.c
index 1a8cdf9..6c67cc4 100644
--- a/drivers/acpi/acpi_memhotplug.c
+++ b/drivers/acpi/acpi_memhotplug.c
@@ -152,8 +152,9 @@ static int acpi_memory_check_device(struct acpi_memory_device *mem_device)
  unsigned long long current_status;
 
  /* Get device present/absent information from the _STA */
- if (ACPI_FAILURE(acpi_evaluate_integer(mem_device->device->handle, "_STA",
-            NULL, &current_status)))
+ if (ACPI_FAILURE(acpi_evaluate_integer(mem_device->device->handle,
+            METHOD_NAME__STA, NULL,
+            &current_status)))
   return -ENODEV;
  /*
   * Check for device status. Device should be
-- 
1.7.1